Update Element Timing API pages (#22846)
mdn/content: files/en-us/web/api/element/elementtiming/index.md

---
title: Element.elementTiming
slug: Web/API/Element/elementTiming
page-type: web-api-instance-property
browser-compat: api.Element.elementTiming
---

{{DefaultAPISidebar("DOM")}}

> The **`elementTiming`** property of the {{domxref("Element")}} interface reflects the value of the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute which marks an element for observation in the {{domxref("PerformanceElementTiming")}} API.

Reviewer: This is a bit hard to parse, should we try to restructure? I'm not sure if this is an improvement:

```suggestion
The **`elementTiming`** property of the {{domxref("Element")}} interface identifies elements for observation in the {{domxref("PerformanceElementTiming")}} API. The `elementTiming` property reflects the value of the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute.
```

Author: Yes, I think this is better. I don't know if we have a standard wording for HTML content attribute reflection and what it means. Maybe something to standardize some day.

## Value

A string.

## Examples

### Logging the value of `elementTiming`

> In this example an {{HTMLElement("image")}} element is set to being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute.

Reviewer 2: ```suggestion
In this example, adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute to the {{HTMLElement("img")}} element sets the image to be observed.
```

Reviewer 2: alternatively, 

```
In this example, adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute to the {{HTMLElement("img")}} element sets the image to be observed by the Element Timing API.
```

```html
<img
  src="image.jpg"
  alt="a nice image"

>   elementtiming="big-image"

Reviewer 2: should we add this attribute to the [`<imag>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) element? It's on the sidebar but not in the text.

Author: Added in https://github.com/mdn/content/pull/22846/commits/a28da1661a63cc92330f8bcf281c7aef14269d42

  id="myImage" />
```

> You can get the string provided to the `elementtiming` HTML attribute by calling `el.elementTiming`.

Reviewer 2: ```suggestion
You can get the string value of the `elementtiming` HTML attribute by calling `el.elementTiming`.
```
